pages/common/perl.md

```diff
@@ -21,3 +21,11 @@
 - Run a Perl script in debug mode, using `perldebug`:
 
 `perl -d {{script.pl}}`
+
+- Replace all occurrences of a string in a file with another string, overwriting the file in-place:
+
+`perl -p -i -e 's/{{find}}/{{replace}}/g' {{filename}}`
```

Comment:
@morrme yes, I wondered if we could somehow squeeze the description of "p" into something like "pack" (I couldn't come up with another stand-in for "wrap" or "loop" including "p"). But I am also not really sure what this loop means, and what would happen without it. Probably understanding this mechanism better would help describing it :)